### PR TITLE
feat: ci setup

### PR DESCRIPTION
**Pull request by maddhruv**
Add CI Workflows for:
- lint
- test

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: test
```
Comment (rajanand02) [inline .github/workflows/test.yml]: why two jobs?

Comment (maddhruv) [reply .github/workflows/test.yml]: just segregating concerns of CI

File: .github/workflows/lint.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '13.x'
+
```
Comment (rajanand02) [inline .github/workflows/lint.yml]: add caching

File: .github/workflows/test.yml (hunk 1 above)
Comment (rajanand02) [reply .github/workflows/test.yml]: Please keep it in a single job

**Pull request merged**